example/src/App.tsx

```diff
@@ -27,6 +27,11 @@ const App = () => {
     android: 'ENTER_ANDROID_REWARDED_AD_UNIT_ID_HERE',
   });
 
+  const MREC_AD_UNIT_ID = Platform.select({
```

Comment:
Can we declare this under L 38 since banner buttons/info are technically declared/shown first